views/js/controller/creator/index.js

```diff
@@ -47,7 +47,7 @@ define([
             'menu':              $('.menu', $container),
             'menuLeft':          $('.menu-left', $container),
             'menuRight':         $('.menu-right', $container),
-            'title':             $('#item-editor-panel > h1', $container),
+            'title':             $('#item-editor-panel .toolbar-container h1', $container),
```

Comment:
I'm not sure here: why is the title in a toolbar container and the toolbar in the title area ?
The toolbar should be in a 'toolbar' area and the title in a 'title' area. 

Sure we may want to hide the title area when the toolbar is displayed but this is a different concern that the area broker should not be aware of.